2-library/6core-logging/src/main/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <root level="DEBUG">
```

Comment:
I think we should also save some logs (+date and line number of where log was occurred) into file, just for being able to find some bugs not in debug mode.

Reply:
yes, you are right

Reply:
Here is nice option for pattern:
```
<pattern>
                <![CDATA[%clr(%d{yyyy-MM-dd HH:mm:ss.SSS}){faint} %clr(%5p) %clr([${appName:-unknown}]){magenta} %clr([traceId: %X{traceId:-n/a}]){magenta} %clr([spanId: %X{spanId:-n/a}]){magenta} %clr([pid: ${PID:- }]){magenta} %clr(---){faint} %clr([%15.15t]){faint} %clr(%-40.40logger{39}){cyan} %clr(:){faint} %m%n%wEx]]>
</pattern>
```

Reply:
ok, but I used it without colors